.github/workflows/ansible-test.yml

```diff
@@ -80,18 +80,26 @@ jobs:
           - stable-2.10
           - devel
         python:
+          - 2.7
+          - 3.5
           - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+        exclude:
+          - ansible: stable-2.9
+            python: 3.9
 
     steps:
       - name: Check out code
         uses: actions/checkout@v2
         with:
           path: ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
 
-      - name: Set up Python ${{ matrix.ansible }}
+      - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python }}
+          python-version: 3.8
```

Comment:
Should these be set back to the matrix values now?

Reply:
No. It should have never been the matrix value. (https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml#L161)

Reply:
Ohhh interesting, so how does the right python version get into place for `ansible-test` to use? Is it that the default container just has all of them?

Reply:
The default container has all of them. Or well, almost ;) Ansible 2.9's ansible-test only has Pythons up to Python 3.8, while 2.9 and devel also support Python 3.9. That's why there's the `exclude` entry in the matrix.